contracts/token/ERC1155/ERC1155.sol

```diff
@@ -165,25 +218,9 @@ contract ERC1155 is Context, ERC165, IERC1155, IERC1155MetadataURI {
         bytes memory data
     ) internal virtual {
         require(to != address(0), "ERC1155: transfer to the zero address");
```

Comment:
As in ERC20, we need to check that `from` isn't 0 either.

Reply:
Since the check wasn't there before the change, I wasn't sure if it  was needed here, but it makes sense that after this change we want to make sure that the transfer is called only for transfers. Therefore, either `from` or `to` can be the zero address.